stdlib/websub/src/main/ballerina/src/websub/publisher_client.bal

```diff
@@ -57,10 +63,13 @@ public type PublisherClient client object {
         }
     }
 
-    # Unregisters a topic in a Ballerina WebSub Hub.
-    #
-    # + topic - The topic to unregister
-    # + return - `error` if an error occurred unregistering the topic
+# Unregisters a topic in a Ballerina WebSub Hub.
+# ```ballerina
+# error? unregisterTopic = websubHubClientEP->unregisterTopic("http://websubpubtopic.com");
+#  ```
+#
+# + topic - The topic to unregister
+# + return -  An `error`if an error occurred unregistering the topic or else `()`
```

Comment:
```suggestion
# + return -  An `error`if an error occurred unregistering the topic or else `()`
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.